src/qrc.ml

```diff
@@ -182,17 +182,15 @@ module Matrix = struct
     let on, off = if invert then "white", "black" else "black", "white" in
     let ( ++ ) acc d = d :: acc in
     let acc =
-      [] ++ "<svg xmlns='http://www.w3.org/2000/svg' \
-             xmlns:l='http://www.w3.org/1999/xlink' version='1.1' " ++
-             "width='" ++ w_mm ++ "mm' " ++ "height='" ++ w_mm ++ "mm' " ++
-             "viewBox='0 0 " ++ w ++ " " ++ w ++ "'>\n" ++
-      " <defs><g id='o'><rect width='1.05' height='1.05' fill='" ++ on ++
-      "'/></g></defs>\n" ++
-      " <rect width='" ++ w ++ "' height='" ++ w ++ "' fill='" ++ off ++ "'/>"
+      [] ++ "<svg xmlns='http://www.w3.org/2000/svg' version='1.1' " ++
+      "width='" ++ w_mm ++ "mm' " ++ "height='" ++ w_mm ++ "mm' " ++
+      "viewBox='0 0 " ++ w ++ " " ++ w ++ "'>\n" ++
+      " <rect width='" ++ w ++ "' height='" ++ w ++ "' fill='" ++ off ++ "'/>" ++
+      " <path style='fill:'" ++ on ++ "' d='"
```

Comment:
There's a spurious `'` after the `fill:` but nevermind I'll amend your commit.

Reply:
Also any particular reason why you used `style` rather than the `fill` attribute ?

Reply:
No reason, I just took inspiration from the code generated by  https://api.qrserver.com/v1/create-qr-code/?size=150x150&data=Example&format=svg.